### PR TITLE
Update characteristics.stories.mdx

### PR DESCRIPTION
**Pull request by sion-docs**
Minor text updates throughout

### CODE REVIEW

File: packages/theme/stories/characteristics/characteristics.stories.mdx [hunk 1]
```diff
@@ -9,24 +9,24 @@ import "./characteristics.stories.css";
...
 
 ### Why characteristics?
 
-From a design perspective, there is a need for aesthetic flexibility without decreasing the efficiency of build or the effectiveness of the components when implemented. From a dev perspective, it provides an easy way to reuse the same values, without having to remember pesky hex values, for example. Any time we want to tweak the value of a characteristic, we only have to do it in one single place for it to reflect everywhere.
+From a design perspective, there is a need for aesthetic flexibility without decreasing the efficiency of build or the effectiveness of the components when implemented. From a dev perspective, it provides an easy way to reuse the same values, without the overhead of having to remember details such as hex values. Any time you want to tweak the value of a characteristic, you only have to do it in one place for it to reflect everywhere.
 
-Characteristics give us:
+Characteristics offer the following benefits:
 
 - Ability to theme the UITK inline with third-parties or CIB sub brand guidelines
```
Comment (joshwooding) [inline packages/theme/stories/characteristics/characteristics.stories.mdx:24]: Do these bullet points need full stops?

Comment (sion-docs) [reply packages/theme/stories/characteristics/characteristics.stories.mdx]: Yes, fixed

**Review by libertymayc — APPROVED** [review]

File: packages/theme/stories/characteristics/characteristics.stories.mdx [hunk 2]
```diff
@@ -141,7 +142,7 @@ Let's add the following to `Card.css`:
 }
 ```
 
-Note how we make use of the `-high` emphasis characteristic tokens provided by UITK that weren't used in Card by default as well as raw values to further emphasise the component. Now, all we have to do to make Card have emphasis is pass in the `uitkEmphasisHigh` prop:
+Note the use of `-high` emphasis characteristic tokens provided by UITK that weren't used in Card by default, as well as raw values to further emphasise the component. Now, all we have to do to make Card have emphasis is pass in the `uitkEmphasisHigh` prop:
```
Comment (Fercas123) [inline packages/theme/stories/characteristics/characteristics.stories.mdx:145]:
```suggestion
Note the use of `-high` emphasis characteristic tokens provided by UITK that weren't used in Card by default, as well as raw values to further emphasize the component. Now, all we have to do to make Card have emphasis is pass in the `uitkEmphasisHigh` prop:
```

File: packages/theme/stories/characteristics/characteristics.stories.mdx [hunk 3]
```diff
@@ -141,7 +142,7 @@ Let's add the following to `Card.css`:
 }
 ```
 
-Note how we make use of the `-high` emphasis characteristic tokens provided by UITK that weren't used in Card by default as well as raw values to further emphasise the component. Now, all we have to do to make Card have emphasis is pass in the `uitkEmphasisHigh` prop:
+Note the use of `-high` emphasis characteristic tokens provided by UITK that weren't used in Card by default, as well as raw values to further emphasize the component. Now, all we have to do to make Card have emphasis is pass in the `uitkEmphasisHigh` prop:
```
Comment (joshwooding) [inline packages/theme/stories/characteristics/characteristics.stories.mdx:145]:
```suggestion
Note the use of `-high` emphasis characteristic tokens provided by UITK that weren't used in Card by default, as well as raw values to further emphasize the component. Now, all we have to do to make Card have emphasis is pass in the `uitkEmphasisHigh` class name:
```

**Review by joshwooding — APPROVED** [review]

**Pull request merged**